### ISSUE TITLE
@svg-icons/fa-brands

### ISSUE THREAD

**Issue by ChiefORZ** [webpro/knip#322]
Importing files from the dependency **@svg-icons/fa-brands** and **@svg-icons/heroicons-outline** are now detected as **Unused dependencies** since the version _2.38.2_, worked fine up till version _2.38.1_

<img width="348" alt="image" src="https://github.com/webpro/knip/assets/6916518/e121c267-f2d6-4141-a78a-9c16b2d02866">

```
import Github from '@svg-icons/fa-brands/github.svg';
import LinkedinIn from '@svg-icons/fa-brands/linkedin.svg';
import Twitter from '@svg-icons/fa-brands/twitter.svg';
import Briefcase from '@svg-icons/heroicons-outline/briefcase.svg';
import Envelope from '@svg-icons/heroicons-outline/envelope.svg';
import GlobeAlt from '@svg-icons/heroicons-outline/globe-alt.svg';
import Home from '@svg-icons/heroicons-outline/home.svg';
import LinkIcon from '@svg-icons/heroicons-outline/link.svg';
import Printer from '@svg-icons/heroicons-outline/printer.svg';
import User from '@svg-icons/heroicons-outline/user.svg';
```

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v2.38.5. See [Release 2.38.5](https://github.com/webpro/knip/releases/tag/2.38.5) for release notes._

**Comment by webpro** [comment]
Thanks for reporting this, @ChiefORZ. I've added more fixtures/coverage and expect this to give less issues going forward.
